validator/client/src/test/java/tech/pegasys/teku/validator/client/loader/ValidatorLoaderTest.java

```diff
@@ -303,7 +304,7 @@ void initializeValidatorsWithDuplicateKeysInLocalAndExternalSignersTakesExternal
   void shouldEnableSlashingProtectionForLocalValidators(@TempDir Path tempDir) throws Exception {
     writeKeystore(tempDir);
 
-    final InteropConfig interopConfig = InteropConfig.builder().build();
+    final InteropConfig interopConfig = InteropConfig.builder().specProvider(specProvider).build();
```

Comment:
nit: Probably should have done this way back but might be worth extracting this as a `disabledInteropConfig` field to avoid the duplication.